src/pages/index.js

```diff
@@ -9,13 +9,51 @@ import {initialCards, selectors, validationSettings, cardAddForm, profileEditFor
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from '../components/UserInfo.js';
+import Api from "../components/Api.js";
+import { PopupWithConfirmation } from '../components/popupWithConfirmation.js';
```

Comment:
```js
from '../components/popupWithConfirmation.js';
```


You have `PopupWithConfirmation.js`, so the import should be like this


```js
import { PopupWithConfirmation } from '../components/PopupWithConfirmation.js';


```

Reply:
* One thing hasn't been corrected https://github.com/Safiul-Alam/se_project_aroundtheus/pull/1#discussion_r1797584587